content/pages/conference/2024/06-seminar.rst

```diff
@@ -0,0 +1,108 @@
+====================================================
+sciwork seminar summer 2024
+====================================================
+
+:date: 2024-05-09 21:00
+:url: seminar/2024/06-seminar
+:save_as: seminar/2024/06-seminar.html
+
+Date
+----
+
+* Date: 22nd June, Saturday, 2024
+* Time: 8:30 -- 17:40
+
+|
+
+Talks in seminar
+----------------
+
+* Visualization that balances performance and flexibility for scientific computing by **Chun-Hsu Lai**
```

Comment:
Confirmed topics should be made explicit and separate from unconfirmed ones.  Check the workbook.